src/Modules/CalcOffence.lua

```diff
@@ -1549,41 +1549,32 @@ function calcs.offence(env, actor, activeSkill)
 			output.TriggerTime = output.Time
 			output.Speed = 1 / output.Time
 		elseif skillData.triggerRate and skillData.triggered then
-			-- Account for trigger unleash
-			if skillData.triggerUnleash then
-				-- process the source trigger skill to get it's full data
-				local calcMode = env.mode == "CALCS" and "CALCS" or "MAIN"
-				for _, triggerSkill in ipairs(actor.activeSkillList) do
-					if cacheSkillUUID(triggerSkill) == skillData.triggerSourceUUID then
-						calcs.buildActiveSkill(env, calcMode, triggerSkill)
-						break
-					end
-				end
-				local cachedSourceSkill = GlobalCache.cachedData[calcMode][skillData.triggerSourceUUID]
-				-- if properly processed, get it's dpsMultiplier to increase triggerRate
-				if cachedSourceSkill then
-					skillData.unleashTriggerRate = skillData.triggerRate * (cachedSourceSkill.ActiveSkill.skillData.dpsMultiplier or 1)
-					if breakdown then
-						breakdown.Speed = {
-							s_format("%.2f ^8(trigger rate)", skillData.triggerRate),
-							s_format("* %.2f ^8(multiplier from Unleash)", cachedSourceSkill.ActiveSkill.skillData.dpsMultiplier or 1),
-							s_format("= %.2f", skillData.unleashTriggerRate),
-						}
-					end
-					-- over-write the triggerRate modifier after breakdown as other calcs use it
-					skillData.triggerRate = skillData.unleashTriggerRate
-				end
-				-- give this activeSkill "HasSeals" flag so Configuration Option for UseMaxUnleash is available
-				activeSkill.skillFlags.HasSeals = true
-			end
 			output.Time = 1 / skillData.triggerRate
 			output.TriggerTime = output.Time
 			output.Speed = skillData.triggerRate
 			skillData.showAverage = false
 		elseif skillData.triggeredByBrand and skillData.triggered then
-			output.Time = 1 / (1 + skillModList:Sum("INC", cfg, "Speed", "BrandActivationFrequency") / 100) / skillModList:More(cfg, "BrandActivationFrequency") * (skillModList:Sum("BASE", cfg, "ArcanistSpellsLinked") or 1)
+			local activationFreqInc = 100 / (100 + skillModList:Sum("INC", cfg, "Speed", "BrandActivationFrequency"))
+			local activationFreqMore = skillModList:More(cfg, "BrandActivationFrequency")
```

Comment:
This was divided in the original formula which is correct? (more)

Reply:
I've broken the formula down a bit here. division = multiplication by the inverse. It _should_ produce the same values as before. 
![obraz](https://user-images.githubusercontent.com/91493239/205622844-75669c74-2bd6-493e-9ab2-166d0402009c.png)

Reply:
44% more activations => smaller time

Reply:
Time is 1/1.44  = 0.69 here. So that checks out ? Or should it be the other way around ? In release it's it's just 1.44 so i guess it should be time = 1.44 without the the inverse.

Reply:
I think i had it the other way around. 
![obraz](https://user-images.githubusercontent.com/91493239/205625896-0a377d47-8b2c-4b3d-b9f0-26f0c995bfe7.png)

Reply:
Looks good now just got to fix the breakdown.

Reply:
![obraz](https://user-images.githubusercontent.com/91493239/205627989-faefa160-5998-4028-948d-a17c97ab80f7.png)